companies.html

```diff
@@ -488,6 +488,17 @@
       Uses Scrapy to crawl and scrape taxi and transportation sites to gather rates, coverage zones, and deals.</p>
     </div>
 
+    <div class="company-box">
+      <a href="https://www.allogarage.fr/">
```

Comment:
Hello,

There’s a mistake, it’s http, not https sorry
Here is the logo, I can’t find a way to upload an img (don’t have write access…).

And I’ve just tweeted :)


Best Regards,


Nicolas

Le 9 juin 2016 à 11:26, Paul Tremberth <notifications@github.com> a écrit :

> In companies.html:
> 
> > @@ -488,6 +488,17 @@
> >        Uses Scrapy to crawl and scrape taxi and transportation sites to gather rates, coverage zones, and deals.</p>
> >      </div>
> >  
> > +    <div class="company-box">
> > +      <a href="https://www.allogarage.fr/">
> https:// version does not load for me.
> 
> Also, is there any chance of you blogging or tweeting about how you use scrapy?
> That would be great
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>